src/tools/sssctl/sssctl_config.c

```diff
@@ -75,6 +75,11 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     struct poptOption long_options[] = {
         {"config", 'c', POPT_ARG_STRING, &config_path,
             0, _("Specify a non-default config file"), NULL},
+        {"snippet", 's', POPT_ARG_STRING, &config_snippet_path,
+            0, _("Specify a non-default snippet dir (The default is to look at "
+                 "the same place where main config file is located. For "
+                 "example if config is set to \"/my/path/sssd.conf\", "
+                 "snippet dir \"/my/path/conf.d\" is used)"), NULL},
```

Comment:
A few very minor suggestions:
* "look at" to "look in"
* "where main config file is" to "where the main config file is"
* "if config is set" to "if the config is set"
* "snippet dir" to "the snippet dir"

Reply:
Thanks for the review, fixed